.github/workflows/CI.yml

```diff
@@ -10,6 +10,8 @@ on:
       - "release-"
     tags: "*"
   pull_request:
+  # to allow manual tests https://leonardomontini.dev/github-action-manual-trigger/
```

Comment:
nit: "to allow manual tests: https://leonardomontini.dev/github-action-manual-trigger/"